src/main/java/com/coveros/selenified/application/App.java

```diff
@@ -744,6 +746,35 @@ public void selectFrame(String frameIdentifier) {
         file.recordAction(action, expected, expected, Result.SUCCESS);
     }
 
+    /**
+     * Safari, Edge and IE don't recognize the allowInsecureCert capability. As a result, an alternative method is
+     * provided in order to 'manually' accept the problematic certificate
+     */
+    public void acceptCertificate() {
+        String action = "Clicking override link to accept ssl certificate";
+        String result = "Override link clicked";
+        //for IE and Edge
+        if (browser.getName() == BrowserName.INTERNETEXPLORER || browser.getName() == BrowserName.EDGE) {
+            Element overrideLink = newElement(Locator.ID, "overridelink");
+            if (overrideLink.is().present()) {
+                try {
+                    if (browser.getName() == BrowserName.EDGE) {
+                        newElement(Locator.ID, "moreInformationDropdownSpan").getWebElement().click();
+                    }
+                    overrideLink.getWebElement().click();
+                    file.recordAction(action, result, result, Result.SUCCESS);
+                } catch (Exception e) {
+                    file.recordAction(action, result, "Unable to click override link. "
+                            + e.getMessage(), Result.FAILURE);
+                    file.addError();
+                    log.warn(e);
+                }
+            }
+        }
+        //for Safari
+        //TODO, pending sauce ticket #63099
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1135)